cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -90,49 +109,48 @@
                ng-click="LogViewer.toggleLogExpansion()">Expand All</div>
         </th>
       </thead>
-
       <tbody>
         <tr ng-repeat="log in LogViewer.data | filter:LogViewer.filterByStartDate | filter:LogViewer.eventFilter | filter:searchText">
           <!-- TIME -->
           <td class="time"
               ng-if="LogViewer.configOptions.time && !LogViewer.hiddenColumns.time">
-            <div class="table-text-content">{{ ::log.displayTime }}</div>
+            <div class="table-text-content">{{ ::log.log.displayTime }}</div>
           </td>
 
           <!-- LEVEL -->
-          <td class="level"
+          <td class="level2"
               ng-if="LogViewer.configOptions.level && !LogViewer.hiddenColumns.level">
-            <div class="table-text-content">{{ ::log.level }}</div>
+            <div class="table-text-content">{{ ::log.log.logLevel }}</div>
           </td>
 
           <!-- SOURCE -->
           <td class="source"
               ng-if="LogViewer.configOptions.source && !LogViewer.hiddenColumns.source">
-            <div class="table-text-content">{{ ::log.source }}</div>
+            <div class="table-text-content">{{ ::log.log.threadName }}</div>
           </td>
 
           <!-- MESSAGE -->
           <td class="message"
               ng-if="LogViewer.configOptions.message"
-              ng-class="{'stack-trace-expanded': log.message.stackTrace && log.isStackTraceExpanded,
-                'stack-trace-available': log.message.stackTrace }">
+              ng-class="{'stack-trace-expanded': log.log.stackTrace.length > 0 && log.isStackTraceExpanded,
```

Comment:
is there a case where ```log.isStackTraceExpanded``` is true but ```stackTrace.length``` is 0? 

log.isStackTraceExpanded should only be set if there is stackTrace right?